src/components/steps/AssertionStep.vue

```diff
@@ -0,0 +1,38 @@
+<template>
+  <div class="step assert">
+
+    <i class="far fa-eye" v-if="!isNegative(step.name)"></i>
```

Comment:
Why did you bring the icons back? I thought you said you find them unintuitive.

Reply:
Ha-ha, I took a second round to think 😃
Well, if there is a common principle, why not to use icons if they save space and focus.

So I decided to use icons for waiters, grabbers, assertions.
As well as custom styles for those step types.

Reply:
I think my initial dislike was based on the fact that each step had its own icon. Which was not very clear - which one have what icon and on which principle they are applied